lib/open_pgp/public_key_encrypted_session_key_packet.ex

```diff
@@ -156,4 +165,61 @@ defmodule OpenPGP.PublicKeyEncryptedSessionKeyPacket do
       raise(msg)
     end
   end
+
+  @doc """
+  Encrypt session key with a given public key.
+  Require session key algo as it will be encoded and encrypted as well.
+  Require public key algo as it will be used to generate public key material.
+  Return ciphertext binary (encrypted session key), which consist of algorithm specific encrypted MPIs.
+  """
```

Comment:
Most of the other public functions have doctests. Is it possible to be consistent? Would also give reviewers like me (not familiar with the codebase) more context!

Reply:
You are right, however the encryption is hard to test. I've decided to keep it in the test file, as it's pretty complicated. I'll think of something relatively "simple" an concise to show as an example. Thanks for bringing that up.